specs/experimental/op-contracts-manager.md

```diff
@@ -20,146 +20,247 @@ of governance approved [contract releases] can be found on the
+    function blueprints() external view returns (Blueprints memory);
+    /// @notice Maps an L2 chain ID to an L1 batch inbox address
+    function chainIdToBatchInboxAddress(uint256 _l2ChainId) external pure returns (address);
+    /// @notice Addresses of the latest implementation contracts.
+    function implementations() external view returns (Implementations memory);
+    /// @notice L1 smart contracts release deployed by this version of OPCM.
+    function l1ContractsRelease() external view returns (string memory);
+    /// @notice Address of the ProtocolVersions contract shared by all chains.
+    function protocolVersions() external view returns (address);
+    /// @notice Address of the SuperchainConfig contract shared by all chains.
+    function superchainConfig() external view returns (address);
+    /// @notice Maps an L2 Chain ID to the SystemConfig for that chain.
+    function systemConfigs(uint256) external view returns (address);
+    /// @notice Semver version specific to the OPContractsManager
+    function version() external view returns (string memory);
+
+/// @notice Returns the implementation contract addresses.
+function implementations() public view returns (Implementations memory);
 
-### `deploy`
+/// @notice Maps an L2 Chain ID to the SystemConfig address for that chain.
+/// @notice All contracts for a chain can be found from its SystemConfig.
+function systemConfigs(uint256 chainId) external view returns (SystemConfig);
 
-The `deploy` method is the only non-view method in the contract. It is used to
-deploy the full set of L1 contracts required to setup a new OP Stack chain that
-complies with the [standard configuration]. It has the following interface:
+/// @notice Maps an L2 chain ID to an L1 batch inbox address as defined by the standard
+function chainIdToBatchInboxAddress(uint256 _l2ChainId) public pure returns (address);
 
-```solidity
-struct Roles {
-  address proxyAdminOwner;
-  address systemConfigOwner;
-  address batcher;
-  address unsafeBlockSigner;
-  address proposer;
-  address challenger;
-}
+/// @notice Returns the blueprint contract addresses.
+function blueprints() public view returns (Blueprints memory);
+```
+
+## Deployment
+
+### Interface
 
-function deploy(
-  uint256 l2ChainId,
-  uint32 basefeeScalar,
-  uint32 blobBasefeeScalar,
-  Roles roles
-) external returns (SystemConfig)
+Version 1.0.0 of the OP Contracts Manager deploys the `op-contracts/v1.6.0` contracts release,
+and is deployed at `0x9BC0A1eD534BFb31a6Be69e5b767Cba332f14347`. In the future this will
+be tracked in the `superchain-registry`.
+
+#### `deploy`
+
+The `deploy` method is used to deploy the full set of L1 contracts required to setup a new OP Stack
+chain that complies with the [standard configuration]. It has the following interface:
+
+```solidity
+/// @notice Deploys a new OP Chain
+/// @param _input DeployInput containing chain specific config information.
```

Comment:
What is the definition of `DeployInput`?

Reply:
It's [large](https://github.com/ethereum-optimism/optimism/blob/upgrade-against-registry/packages/contracts-bedrock/src/L1/OPContractsManager.sol#L46) and will change everytime a new configurable state var is added. 

I left it out to minimize the maintenance burden of this file.

Reply:
Addressed: e1c854188bee